GeoFire/API/GFQuery.h

```diff
@@ -28,7 +28,8 @@
 
 #import <Foundation/Foundation.h>
 #import <CoreLocation/CoreLocation.h>
-#import <Firebase/Firebase.h>
+
+typedef NSUInteger FirebaseHandle;
```

Comment:
I don't really like the redefinition here, but I guess that's what you get when do you weird stuff like using typedefs...